package.json

```diff
@@ -5,11 +5,15 @@
   "main": "src/index.js",
   "dependencies": {
     "body-parser": "^1.15.2",
+    "connect-mongodb-session": "^1.1.1",
     "cors": "^2.7.1",
     "express": "^4.14.0",
+    "express-session": "^1.14.0",
     "mongodb": "^2.2.6",
     "mongoose": "^4.5.9",
     "optimist": "^0.6.1",
+    "passport": "^0.3.2",
+    "passport-github": "^1.1.0",
```

Comment:
the passport docs link to passport-github2 as the current implementation, do we want to use that instead?